source/ncconvert/builder/Manifest.cpp

```diff
@@ -59,13 +60,13 @@ auto IsUpToDate(const nc::convert::Target& target) -> bool
     return std::filesystem::last_write_time(target.destinationPath) > std::filesystem::last_write_time(target.sourcePath);
 }
 
-auto ToAssetSubtype(std::string_view str) -> nc::convert::AssetSubtype
+auto ToAssetSubtype(std::string_view str) -> nc::asset::AssetSubtype
 {
-    using enum nc::convert::AssetSubtype;
-    if (str == "diffuse")  return DiffuseTexture;
+    using enum nc::asset::AssetSubtype;
     if (str == "normal")   return NormalTexture;
-    if (str == "particle") return ParticleTexture;
-    if (str == "effect")   return EffectTexture;
+    if (str == "diffuse")  return DiffuseTexture;
```

Comment:
'diffuse' still works a a subdirectory for the textures that are not normal, particle or effect. But the commonality between diffuse, particle and effect is the texture format